tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -215,6 +215,49 @@ stages:
             - sign_windows_fxdependent_release
             - sign_windows_fxdependentWinDesktop_release
 
+- stage: PublishSymbol
+  displayName: Publish symbols
+  dependsOn: Build
+  jobs:
+  - job: Symbols_Job
+    pool:
+      name: PowerShell1ES
+      demands:
+      - ImageOverride -equals PSMMS2019-Secure
+    steps:
+    - download: current
+      artifact: results
+      patterns: '**/*.zip'
+
+    - pwsh: |
+        Write-Verbose -Verbose "Enumerating $(Pipeline.Workspace)/results"
+        $downloadedArtifacts = Get-ChildItem -Recurse "$(Pipeline.Workspace)/results"
+        $downloadedArtifacts
+        $expandedRoot = New-Item -Path "$(Pipeline.Workspace)/expanded" -ItemType Directory -Verbose
+        $symbolsRoot = New-Item -Path "$(Pipeline.Workspace)/symbols" -ItemType Directory -Verbose
+        $downloadedArtifacts | Where-Object { $_.Name -like 'PowerShell*-symbols*-signed.zip'} | ForEach-Object {
```

Comment:
I don't think this package has symbols.

We remove them here:
https://github.com/PowerShell/PowerShell/blob/9d8cc0c62d096567789768e03f164fa45d54cb7b/tools/packaging/packaging.psm1#L760-774

The entry point to all of windows packaging is here:
https://github.com/PowerShell/PowerShell/blob/9d8cc0c62d096567789768e03f164fa45d54cb7b/tools/releaseBuild/azureDevOps/templates/windows-hosted-build.yml#L67

We have code to generate a pdb package here:
https://github.com/PowerShell/PowerShell/blob/9d8cc0c62d096567789768e03f164fa45d54cb7b/tools/packaging/packaging.psm1#L344